### PR TITLE
Fixes #25392 - refresh action test fix

### PR DESCRIPTION
**Pull request by jlsherrill**
caused by two colliding commits

### CODE REVIEW

File: app/lib/actions/pulp/repository/refresh.rb [hunk 1]
```diff
@@ -7,8 +7,8 @@ class Refresh < Pulp::Abstract
           param :pulp_id
         end
 
-        def plan(repository, input = {})
-          plan_self(:capsule_id => input[:capsule_id], :pulp_id => repository.pulp_id)
+        def plan(repository, options = {})
+          plan_self(:capsule_id => options[:capsule_id], :pulp_id => repository.pulp_id)
```
Comment (akofink) [inline app/lib/actions/pulp/repository/refresh.rb]: This change doesn't do anything, right?

Comment (parthaa) [reply app/lib/actions/pulp/repository/refresh.rb]: well input is an instance variable in a plan. Don't want params called the same. More a precautionary thing I guess.

Comment (jlsherrill) [reply app/lib/actions/pulp/repository/refresh.rb]: correct, but input here is overriding the action's input method.

Comment (jlsherrill) [reply app/lib/actions/pulp/repository/refresh.rb]: so i renamed it for clarity

**Pull request merged**